.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thanks for creating your first issue :+1: Feel free to use [Slack](https://www.hypertrace.org/get-started) if you just need some quick help or want to chat'
```

Comment:
I am curious about the link. Can't we point directly to slack?

Reply:
that link changes over time (30 days or something).